src/Blog/View/posts.twig

```diff
@@ -15,8 +22,15 @@
 		<h1>Blog</h1>
 		<ul>
 			{% for post in posts %}
-				<li>
-					<a href="/blog/{{ post.id }}">{{ post.title }}</a>
+				<li class="u-margin-medium">
+					<a class="post-box__img" href="/blog/{{ post.id }}"> <img src="{{post.url}}"/> </a>
+					<div class="post-box__description">
+					<a class="post-description__header" href="/blog/{{ post.id }}"> <h2 class="post-description__title"> {{post.title}}</h2> <span class="post-descritpion__date"> {{post.date}} </span> </a>
+				
+					<p class="post-description__preamble"> {{post.preamble}} </p>
```

Comment:
Add "module line-clamp" css selectors to this element.
This group of classes cuts text, and adds three dots to the end.

Reply:
Where exactly should I add it? I tried few options but it doesn't work and I am not sure what should I do.

Reply:
Sorry, I wasn't precise. It should be in `p` element, line 30 in class names group 
```class="post-description__preamble module line-clamp"
```

Reply:
Done